lib/mayaUsd/ufe/wrapGlobal.cpp

```diff
@@ -20,9 +20,9 @@
 using namespace MayaUsd;
 using namespace boost::python;
 
+// clang-format off
 void wrapGlobal()
 {
     def("getMayaRunTimeId", ufe::getMayaRunTimeId);
-
-    def("getUsdRunTimeId", ufe::getUsdRunTimeId);
 }
+// clang-format on
```

Comment:
Added clang-format off/on to stop it from creating a single line out of all of this.